__test__/todo.test.js

```diff
@@ -0,0 +1,36 @@
+import { createTodo, removeTodoItem } from '../src/addRemoveFunctions.js';
+
+describe('DOM Manipulation', () => {
+  it('should add a item to the DOM', () => {
+    // Arrange
+    const initialHTML = '<div id="0" class="todo-item"></div>';
+    document.querySelector('.todo-container .todo-items-container').innerHTML = initialHTML;
+
+    const item = {
+      description: 'this is a second added item',
+      index: 1,
+      completed: false,
+    };
+
+    // Act
+    createTodo(item);
+    const list = document.querySelectorAll('.todo-item');
+    // Assert
+    expect(list).toHaveLength(2);
```

Comment:
- [ ] I believe the list length should be equal to `1` after adding one item to it. Here you are expecting `2` which is very strange, let's fix that.

Reply:
Hi, thank you for the review but I think it should actually expect 2 because on the "Arrange" stage of my test I already have one div with the class `todo-item` which means when I call `createTodo` function I should end up with my HTML having two items. Can you please verify this with me?